docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM rhel7/rhel
```

Comment:
Should this be a CentOS base image considering this is public, and should we push it to the public docker hub repo i.e. docker.io/feedhenry?